File: C#/uri-1017.cs
// URI - 1017: https://www.urionlinejudge.com.br/judge/pt/problems/view/1017

using System;

namespace URI
{
    class Program
    {
        static void Main(string[] args)
        {
            int timeOfTravel = int.Parse(Console.ReadLine());
            int averageSpeed = int.Parse(Console.ReadLine());

            Console.WriteLine(((averageSpeed / 12.0) * timeOfTravel).ToString("F3"));
        }
    }
}

File: JavaScript-Node/uri-1017.js
// URI - 1017: https://www.urionlinejudge.com.br/judge/pt/problems/view/1017

const readline = require('readline')
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
})

rl.on('line', input => {
    const timeOfTravel = parseInt(input)

    rl.on('line', input => {
        const averageSpeed = parseInt(input)

        console.log(((averageSpeed / 12) * timeOfTravel).toFixed(3))

        rl.close()
    })
})